sources/packages/backend/apps/queue-consumers/src/processors/schedulers/base-scheduler.ts

```diff
@@ -47,6 +48,25 @@ export abstract class BaseScheduler<T> implements OnApplicationBootstrap {
    * any old cron job delete it and add the new job to the queue.
    */
   async onApplicationBootstrap(): Promise<void> {
+    // TODO: ConfigService below is not injected since it will be removed when the Fulltime is released.
```

Comment:
The TODO must be towards removing the fulltime allowed logic later. 
And must additionally explain the reason of not injecting the config service.